lib/fog/compute/kubevirt.rb

```diff
@@ -44,9 +71,30 @@ def object_to_hash(object)
 
           result
         end
+
+        # Copied from rails:
+        # File activesupport/lib/active_support/core_ext/hash/deep_merge.rb, line 21
```

Comment:
can we import this somehow? 
If not, if we think we need to know it is copied from active_support, we should have a link to the code here because line numbers change across versions and it is a headache to look when this line was created, check which version of active support was used then etc.

Reply:
No, we need to use use fog-kubevirt outsize of rails. Having a link is very good idea.